lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -304,20 +304,28 @@ class BlockedToMMA : public mlir::RewritePattern {
     } else {
 
       // convert operands
-      int minBitwidth =
-          std::min(computeOrigBitWidth(a), computeOrigBitWidth(b));
-      Type minType = IntegerType::get(ctx, minBitwidth);
+      int aBitWidth = computeOrigBitWidth(a);
+      int bBitWidth = computeOrigBitWidth(b);
+      int minBitwidth = std::min(aBitWidth, bBitWidth);
+      // TODO: remove s8xf32 matmul special case when issue is fixed.
+      // https://github.com/openai/triton/issues/2853
```

Comment:
Thanks for the explanation in the PR.

Could we write an explanation here as well?  If I'm understanding correctly, it seems that this is a workaround to avoid a segfault which happens later in the compiler?

Reply:
Updated, let me know if it is not clear